building/export_plugin_from_eclipse.adoc

```diff
@@ -0,0 +1,37 @@
+= How to Export a plugin from Eclipse
+
+Exporting a plugin from inside eclipse is the 'quick and dirty' method to get an output binary for quick local tests. It does not generate an update site, and so this mechanism is only really suitable for local smoke-testing. 
```

Comment:
what is this for actually ? 
when would one use this instead of just going to target folder and take the build there ? (assuming of course the plugin has a pom.xml) ?